apps/rush/src/taskRunner/ProjectBuildTask.ts

```diff
@@ -151,22 +168,8 @@ export default class ProjectBuildTask implements ITaskDefinition {
           return;
         }
 
-        // Normalize test command step
-        build.args.push(this._errorDisplayMode === ErrorDetectionMode.VisualStudioOnline ? '--no-color' : '--color');
-
-        if (this._production) {
-          build.args.push('--production');
-        }
-        if (this._npmMode) {
-          build.args.push('--npm');
-        }
-        if (this._minimalMode) {
-          build.args.push('--minimal');
-        }
-
-        const actualBuildCommand: string = `${build.command} ${build.args.join(' ')}`;
-
         // Run the test step
+        const actualBuildCommand: string = `${build.command} ${build.args.join(' ')}`;
```

Comment:
it has been working so far... how should it be escaped?

---
In reply to: [116881577](https://github.com/Microsoft/web-build-tools/pull/195#discussion_r116881577) [](ancestors = 116881577)